data.sql

```diff
@@ -3,4 +3,11 @@ VALUES
 ('Agumon', '2020-02-03', 0, true, 10.23),
 ('Gabumon', '2018-11-15', 2, true, 8.00),
 ('Pikachu', '2021-01-07', 1, false, 15.04),
-('Devimon', '2017-05-12', 5, true, 11.00);
\ No newline at end of file
+('Devimon', '2017-05-12', 5, true, 11.00),
+('Charmander', '2020-02-08', 0, false, 11.00),
+('Plantmon', '2021-11-15', 2, true, 5.70),
+('Squirtle', '1993-04-02', 3, false, 12.13),
+('Angemon', '2005-06-12', 1, true, 45.00),
+('Boarmon', '2005-06-07', 7, true, 20.40),
+('Blossom', '1998-10-13', 3, true, 17.00),
+('Ditto', '2022-05-14', 4, true, 22.00);
```

Comment:
Please note that for some of the queries here, the weight you are to give to them should have a negative value. Kindly update the queries as it is good practise to follow instructions 👍